radar/tools/terraform.md

```diff
@@ -5,4 +5,4 @@ quadrant: "tools"
 tags: [cloud, iac, devops]
```

Comment:
It looks like frontmatter is incomplete (we need all tags)

Reply:
what do you mean? Do I need to chose among a set of predefined tags? All tags stands for?

Reply:
sorry I meant that it was missing title and quadrant, but I see it's fixed now. I'd remove the terraform.md from this PR (which is merged into main).